### ISSUE TITLE
Improve readme with project explanation and examples

### ISSUE THREAD

**Issue by bonndan** [DavidBadura/git-webhooks#13]
As far as I understand this library is to receive webhooks? Is Symfony a requirement or can it be used in any php project?

**Comment by tolry** [comment]
@bonndan git-webhooks is a standalone library, not tight to symfony, except that it currently uses a few symfony components and only works with a symfony request object (but there is already an open issue to change this to a psr-7 compliant object, #1).

A README update is definitely in order :smile: - but to give a short explanation here:

git-webhooks normalizes webhooks of different git collaboration platforms, currently only supporting github and gitlab (bitbucket support planned #2).

So, e.g. having a push event from github and gitlab will give you an identical Event value object.

I hope that made things clearer?

**Comment by bonndan** [comment]
Yes, thanks. If I integrate it into a legacy non-sf project every dependency comes out of the box with composer, right?

**Comment by DavidBadura** [comment]
yes, it is a library, not a symfony bundle :wink: the only dependency is symfony/http-foundation and it is in composer.json required.
